src/Control/Monad/Eff/Console/Unsafe.purs

```diff
@@ -0,0 +1,13 @@
+-- | Dodgy functions to output runtime representation of arbitrary values.
```

Comment:
s/dodgy/unsafe maybe? :smile: 

Also not sure about "runtime representation", how about just "display arbitrary values"?